entries/b/binge.com.au.json

```
@@ -0,0 +1,14 @@
+{
+	"Binge": {
+		"domain": "binge.com.au",
+		"contact": {
+			"twitter": "binge"
```

Support / Customer service accounts are generally preferred if available as the chances of the tweets being seen are greater. I think this is their official customer care twitter handle?
```suggestion
			"twitter": "binge_help"
```